src/constants/strings.ts

```diff
@@ -10,4 +10,9 @@ and still undergoing final testing before the official release.
   and any purchasing of storage space should be done
    at the user’s own risk.`
 
+export const API_RESPONSE_MESSAGES = {
+  OK: 'OK',
+  ERROR: 'error',
+}
```

Comment:
Could it be better to have it as
`type ApiResponse = 'OK' | 'ERROR'`?

Reply:
either that or better still,
```suggestion
export const API_RESPONSE_STATUS = {
  OK: 'OK',
  ERROR: 'error',
}

export type ApiResponse = keyof typeof API_RESPONSE_STATUS
```